pageComponents/NewPostPage/PlayerActionSelectors.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import styled from "styled-components";
+import StreetActionSelector from "../../components/StreetActionSelector";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import {
+  GameStreetAction,
+  GameStreetActionType,
+} from "../../models/GameSituation";
+
+interface Props extends React.Attributes {
+  playerActions: {
+    position: string;
+    gameStreetAction: GameStreetAction;
+    tableMaxBetSize: number;
+    previousBetSize: number;
+  }[];
+  onChange?: (
+    index: number,
+    type: GameStreetActionType,
+    betSize: number
+  ) => void;
+}
+
+export default function PlayerActionSelectors({
+  playerActions,
+  onChange = () => undefined,
+}: Props) {
+  return (
+    <PlayerActions>
+      {playerActions.map(
+        ({ position, tableMaxBetSize, previousBetSize }, index) => (
+          <React.Fragment key={index}>
+            <SmallTitle>{`${position}:`}</SmallTitle>
+
+            <div>
+              <StreetActionSelector
+                tableMaxBetSize={tableMaxBetSize}
+                previousBetSize={previousBetSize}
+                onChange={(type, betSize) => {
+                  onChange(index, type, betSize);
+                }}
+              />
+            </div>
+          </React.Fragment>
+        )
```

Comment:
This will render: 

```jsx
<SmallTitle>{...}</SmallTitle>
<div>
  <StreetActionSelector />
</div>
<SmallTitle>{...}</SmallTitle>
<div>
  <StreetActionSelector />
</div>
<SmallTitle>{...}</SmallTitle>
<div>
  <StreetActionSelector />
</div>
// further more ...
```

In my understand, this markup looks quite unnatural because this component should renders lines of label-value pair, so that it will be actually "lines" (such like `<div>`) of those pair: `<div><SmallTitle /><StreetActionSelector /></div>` but it it doesn't as is. Is this your intention?